vault/changelog.md

```diff
@@ -2,10 +2,30 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1663687399589
+updated: 1664203672120
 created: 1601508213606
 nav_order: 2
 ---
+## 0.114.0
+
+### Features
+
+- feat(publish): add configuration option for sidebar (#3448) @sam ([[docs|dendron://dendron.dendron-site/dendron.topic.publish.sidebar]])
+
+### Enhancements
+
+- enhance(publish): upgrade nextjs and various other improvements (#3533) @kevin
+
+### Fix
+
+- fix(publish): click into "empty area" on sidebar submenu item (#3523) @sam
+- fix(publish): incompatibilities with new sidebar implementation (#3542) @sam
+- fix(views): make preview theme background color render correctly (#3552) @sam
+- fix(workspace): Disallow note creation through go to note if filename is invalid (#3551) @hikchoi
+- fix(publish): display headings pleasantly when containing non-text content nodes (#3525) @sam
+- fix(workspace): Update backlinks after engine updates (#3535) @tuling
+
```

Comment:
a new addition to the changelog since early seed: - fix(workspace): engine init with note candidates enabled (#3585) @jonathan